### DISCUSSION TITLE
Increasing max zoom on 2D mode

### DISCUSSION THREAD

**Discussion by aclel** [TerriaJS/terriajs#7280] [Q&A] [answered]
I've not been able to zoom further than level 16 on 2D Mode. I'm able to zoom past 25 on 3D mode.

I've been trying to change `maximumNativeZoom`, `maximumZoom` `maximumLevel` on my layers, but they don't seem to have an effect.

Any ideas?

**Comment by glughi** [comment, marked as answer]
Hello @aclel , you should edit the `layerOptions` in the Leaflet.ts file and add `maxZoom`:

```
const layerOptions = {
  maxZoom: 25,
  bounds: clippingRectangle && rectangleToLatLngBounds(clippingRectangle)
};
```

This change will be used for all layers in Leaflet.

**Reply by aclel** [reply]
Thank you @glughi!
